sycl/test-e2e/Basic/kernel_max_wg_size.cpp

```diff
@@ -5,6 +5,9 @@
 // https://github.com/intel/llvm/blob/sycl/sycl/doc/extensions/experimental/sycl_ext_intel_grf_size.asciidoc
 // REQUIRES: arch-intel_gpu_pvc || gpu-intel-dg2
 // UNSUPPORTED: cuda || hip
+// UNSUPPORTED-INTENDED: This extension is currently implemented in DPC++ only
```

Comment:
Unrelated to the main change in the PR, but tagging @sarnex here. I'm not sure if having an optional property unguarded by an aspect is a good interface for end users. Even if we don't introduce an aspect, can we update the spec to say that submitting a kernel with `grf` property to a device and/or using a backend which doesn't support that results in an exception? This way we would be able to rewrite the test to avoid use of `UNSUPPORTED.

Moreover, we have `REQUIRES` which points to specific GPUs, do we even need `UNSUPPORTED` in that case?